app/routes/bdb/components/bdb.css

```diff
@@ -114,7 +114,7 @@
 }
 
 .interested {
-  background-color: #f5f5d5;
+  background-color: #B33939;
```

Comment:
Can we use a variable instead?😄

Reply:
> Can we use a variable instead?😄

We thought about it. If we only change .interested and not the other classes: interested will change color when we change between light/dark mode, but the others will stay same

Reply:
> > Can we use a variable instead?😄
> 
> We thought about it. If we only change .interested and not the other classes: interested will change color when we change between light/dark mode, but the others will stay same

one solution is to change multiple classes, but then the color at the page will be different and can maybe cause problems for bedroom

Should we still do it?